Bucketeer/Sources/Internal/Model/UserEvaluations.swift

```diff
@@ -1,6 +1,10 @@
 import Foundation
 
-struct UserEvaluations: Codable {
+struct UserEvaluations: Hashable, Codable {
+    // note: should check if we could using `let`
```

Comment:
The id and the evaluations are immutable. So it can be final.

Reply:
tracked in another issue https://github.com/bucketeer-io/ios-client-sdk/issues/25